### PR TITLE
[WIP]: Multi name json input/output

### PR DESCRIPTION
**Pull request by agibsonccc**
Adds multi name input/output endpoint.

### CODE REVIEW

File: konduit-serving-test/src/test/java/ai/konduit/serving/util/SchemaTypeUtilsTest.java [hunk 1]
```diff
@@ -22,16 +22,200 @@
...
+                    break;
+                case Time:
+                    schemaBuilder.addColumnTime(value.name(),TimeZone.getDefault());
+                    break;
+                case Categorical:
+                    schemaBuilder.addColumnCategorical(value.name());
+                    break;
+                case Bytes:
+                    BinaryMetaData binaryMetaData = new BinaryMetaData(value.name());
+                    schemaBuilder.addColumn(binaryMetaData);
```
Comment (ShamsUlAzeem) [inline konduit-serving-test/src/test/java/ai/konduit/serving/util/SchemaTypeUtilsTest.java]: We can probably combine the `Image` and `Bytes` case here

**Comment by ShamsUlAzeem** [comment]
Also, can you write a test for the actual `/dynamicschema` endpoint request?

File: konduit-serving-test/src/test/java/ai/konduit/serving/util/SchemaTypeUtilsTest.java (hunk 1 above)
Comment (agibsonccc) [reply konduit-serving-test/src/test/java/ai/konduit/serving/util/SchemaTypeUtilsTest.java]: I want to keep bytes and images separate here. Technically an ImageWritable type exists and we should figure out how we want to interact with base64/images.

**Comment by agibsonccc** [comment]
@ShamsUlAzeem we have a doJsonInference (the underlying pipeline execution method that gets invoked) test now that tests all types.  I  want to think about the design of the client side experience first as well as see if there's anything else we need to do for the doJsonInference test coverage.

**Pull request merged**